core-kotlin-modules/core-kotlin-flows/pom.xml

```diff
@@ -32,6 +32,20 @@
             <version>3.4.11</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.jetbrains.kotlinx</groupId>
+            <artifactId>kotlinx-coroutines-core</artifactId>
+            <version>1.6.0</version> <!-- Use the latest stable version -->
+        </dependency>
+
+        <!-- Kotlin Coroutines Test -->
+        <dependency>
+            <groupId>org.jetbrains.kotlinx</groupId>
+            <artifactId>kotlinx-coroutines-test</artifactId>
+            <version>1.6.0</version> <!-- Use the latest stable version -->
+            <scope>test</scope>
```

Comment:
Let's pull these versions out into the properties block below, and we probably don't need these comments to use the latest version.